docs/examples/plotting_functions/rainclouds.md

```diff
@@ -0,0 +1,183 @@
+
+# raincloud
+
+{{doc rainclouds}}
+
+\begin{examplefigure}{}
+```julia
+using CairoMakie
+CairoMakie.activate!(type = "png")
+using Random
+using Makie: rand_localized
+
+####
+#### Below is used for testing the plotting functionality.
+####
+
+function mockup_distribution(N)
+    all_possible_labels = ["Single Mode", "Double Mode", "Single Mode", "Double Mode", 
+                           "Random Exp", "Uniform"]
+    category_type = rand(all_possible_labels)
+
+    if category_type == "Single Mode"
+        random_mean = rand_localized(0, 8)
+        random_spread_coef = rand_localized(0.3, 1)
+        data_points = random_spread_coef*randn(N) .+ random_mean
+
+    elseif category_type == "Double Mode"
+        random_mean = rand_localized(0, 8)
+        random_spread_coef = rand_localized(0.3, 1)
+        data_points = random_spread_coef*randn(Int(round(N/2.0))) .+ random_mean
+

```

Comment:
```suggestion
```
potential duplication? otherwise these variables get overridden immediately

Reply:
This looks right to me. (Note the `vcat` on line 33).